java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/appetizer/AppetizerRequestBody.java

```diff
@@ -1,18 +1,13 @@
 package com.my.FoodTruckApp.appetizer;
-//import lombok.*;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-//GENERATE IS A FEATURE OF INTELLIJ
-
-@AllArgsConstructor//annotation does the all args constructor
-@NoArgsConstructor//annotation does the no args constructor
-@Data //has all the getters, setters, and toString built in
-
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
 public class AppetizerRequestBody {
-    private String appSize;
-    private String appFlavor;
-    private String appPairedMeal;
+    private Integer appId;
+    private String appName;
     private Integer appPrice;
```

Comment:
optional: we already know this is an appetizer because of the name `AppetizerRequestBody`. I think it'd be simpler if the fields were direct like `id, name, price` so you don't have to do `appetizerRequestBody.getAppName`.

What do you think?

Reply:
That makes sense, I have updated it